cdc/api/v2/model.go

```diff
@@ -919,6 +919,7 @@ type CSVConfig struct {
 	NullString           string `json:"null"`
 	IncludeCommitTs      bool   `json:"include_commit_ts"`
 	BinaryEncodingMethod string `json:"binary_encoding_method"`
+	EnableOldValue       bool   `json:"enable_old_value"`
```

Comment:
`enable_old_value` has different meanings on different versions of ticdc and has been confusing for users, so a more explicit parameter name such as `output_old_value` is recommended here.

Reply:
`force-split-update` is recommended.

Reply:
output-old-value is more obvious from the semantic perspective, because split-update is just one specific implementation case of output old value in the context of sink to csv files.